package.json

```diff
@@ -33,23 +33,23 @@
         "test:watch": "ts-scripts test --watch asset --"
     },
     "dependencies": {
-        "node-gyp": "11.0.0"
+        "node-gyp": "11.1.0"
```

Comment:
Can you track down and share a link to the changes on `node-gyp`?  This would require a version bump I think, as well as rebuild of the base and Teraslice images right?

Reply:
Link to release notes: https://github.com/nodejs/node-gyp/releases/tag/v11.1.0

Reply:
Since node-gyp is listed as a dependency for the workspace only, not for `terafoundation_kafka_connector`, the version of `node-gyp` used by `node-rdkafka` did not actually update in the `yarn.lock`. `node-gyp@npm:latest` is still at 11.0.0. I think this should be added as a resolution to force the version listed in the root `package.json` to be used.

Reply:
Should the node-gyp dependency be added to the connector instead?

Reply:
`node-gyp` is not a direct dependency in either place. It was originally included with node, but removed in `node v18.18.2`, and something broke, so a top level dependency was added. But I'm not sure why it was necessary to add the dependency, since `node-rdkafka` and several other dependencies include `node-gyp` as a dependency. I need to figure out the 'something' that broke to understand this fully.

Reply:
It looks like we added the top level dependency because we needed to pin the `node-gyp` version. Version 10.0.0 set the minimum node version to 16 and we were still running node 14 tests. I think we should remove it completely and always use the latest version.